### PR TITLE
Feature/automation

### PR DESCRIPTION
**Pull request by spau6**

### CODE REVIEW

**Review by amar-c8y — DISMISSED** [review]

File: cypress-E2E-tests/cypress/integration/asset_properties_widget.cy.ts [hunk 1]
```diff
@@ -584,6 +605,43 @@ describe('Asset Properties Widget: Configuration/View screen tests', function ()
...
+    cy.clickPropertyEditButton('File');
+    cy.get('.dlt-c8y-icon-plus-square').click();
+    selectFile('File', invalidFileSize);
+    verifyFileValidationError(invalidFileSizeErrorMessage);
+    selectFile('File', invalidFileName);
+    verifyFileValidationError(errorMessage);
+    cy.get(editPropertyCancelButton).click();
+    cy.get(asset_properties_widget_elements.settingsButton).click();
+    cy.get(asset_properties_widget_elements.editWidgetButton).click();
+    cy.intercept({ method: 'GET', url: '/inventory/managedObjects/**/childAdditions?pageSize=2000&query=%24filter%3D(has(%27c8y_IsAssetProperty%27))' }).as(
```
Comment (dchc01) [inline cypress-E2E-tests/cypress/integration/asset_properties_widget.cy.ts:626]: An intercept should be added before making the actual call.

File: cypress-E2E-tests/cypress/integration/asset_properties_widget.cy.ts [hunk 2]
```diff
@@ -675,68 +731,12 @@ describe('Asset Properties Widget: Configuration/View screen tests', function ()
     cy.get(assetNameElement).eq(4).should('contain.text', 60);
     cy.get(assetNameElement).eq(5).should('contain.text', 10);
     cy.get(assetNameElement).eq(6).should('contain.text', 15);
+    cy.wait(1000); // Added wait to resolve the flakyness
```
Comment (dchc01) [inline cypress-E2E-tests/cypress/integration/asset_properties_widget.cy.ts]: Verify if adding an intercept solves the issue.

**Review by dchc01 — CHANGES_REQUESTED** [review]

File: cypress-E2E-tests/cypress/integration/asset_properties_widget.cy.ts (hunk 2 above)
Comment (spau6) [reply cypress-E2E-tests/cypress/integration/asset_properties_widget.cy.ts]: Added intercept

File: cypress-E2E-tests/cypress/integration/asset_properties_widget.cy.ts (hunk 1 above)
Comment (spau6) [reply cypress-E2E-tests/cypress/integration/asset_properties_widget.cy.ts:626]: @AmarjyotiSAG please have a look at this

Comment (amar-c8y) [reply cypress-E2E-tests/cypress/integration/asset_properties_widget.cy.ts:626]: incorporated.

**Comment by amar-c8y** [comment]
Sucessful run:- https://github.com/SoftwareAG/cumulocity-asset-properties-widget/actions/runs/7566991566

**Review by amar-c8y — APPROVED** [review]

**Review by dchc01 — APPROVED** [review]

**Pull request merged**